### ISSUE TITLE
StreamHandler Pipe to Process?

### ISSUE THREAD

**Issue by holybit** [Seldaek/monolog#452]
StreamHandler uses [fopen](http://php.net/manual/en/function.fopen.php) which is great for logging to files, but what about logging to another process? For example:

```
$log->pushHandler(new StreamHandler('|/usr/bin/mylogger.py', Logger::DEBUG));
```

Is there a way to do this currently in Monolog?

**Comment by anlutro** [comment]
Have you tried creating a FIFO (named pipe) and use the StreamHandler to write to it?

**Comment by jackminh** [comment]
I want to use monolog to log write redis, there is an error, how can I solve?

**Comment by Seldaek** [comment]
@jackminh first of all you can report a new issue for things that don't have any connection with this one. And then if you have an error you might want to post the error, because otherwise we have no clue how to help.

**Comment by Seldaek** [comment]
Fixed by #720 (only in the upcoming 2.0)
